authentication/views.py

```diff
@@ -1,13 +1,35 @@
-from django.shortcuts import render
+from django.shortcuts import render,redirect
 from django.contrib.auth import login,logout,authenticate
-# Create your views here.
+from django.contrib.auth.forms import UserCreationForm,AuthenticationForm
+from django.contrib.auth.models import User
+from django.db import IntegrityError
+from library.forms import SignUpForm
 
 
 def loginView(request):
-    pass
+    if request.method == 'GET':
+        return render(request,'authentication/login.html',{'form':AuthenticationForm()})
+    else:
+        user = authenticate(request,username=request.POST['username'],password=request.POST['password'])
```

Comment:
You are directly accessing POST data without checking if it even exists. This may lead to server crash if a user access this endpoint with invalid request data. The good behavior would have been to throw a client error (400), rather than server error (500).